src/app/applications/applications-menu/applications-menu.component.html

```diff
@@ -23,19 +24,24 @@
       </a>
     </div>
 
-    <md-divider *ngIf="applicationsData.recents?.length > 0"></md-divider>
+    <div *ngIf="applicationsData.favorites?.length > 0" class="mat-divider"></div>
 
     <div class="cbp-menu-item-category" *ngIf="applicationsData.favorites?.length > 0">Favorite Apps</div>
     <div class="mat-menu-item" *ngFor="let app of applicationsData.favorites">
-      <div class="cbp-applications-remove-icon"><md-icon  fontSet="fontawesome" fontIcon="fa-times" (click)="removeFromFavorite(app, $event)" fxFlexAlign="center"></md-icon></div>
       <a class="cbp-applications-link"  title="{{app.description}}" href="{{app.href}}" fxFlex="" >
         {{app.name}}
       </a>
     </div>
 
-    <md-divider *ngIf="applicationsData.favorites.length > 0 || applicationsData.recents.length > 0"></md-divider>
 
-    <div class="mat-menu-item cbp-applications-refresh" (click)="reloadApplicationsData($event)">Refresh
+    <div class="mat-divider" *ngIf="applicationsData.favorites?.length > 0 || applicationsData.recents?.length > 0"></div>
+    <cbp-applications-search></cbp-applications-search>
+
+
+    <div class="mat-menu-item cbp-applications-refresh" (click)="reloadApplicationsData($event)">
```

Comment:
Is this accessible?

Reply:
Missed your comments. `<cbp-applications-search></cbp-applications-search>` is probably as accessible as the material components it is composed off.